src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,58 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int CIRCLE_RADIUS = 10;
+    public static final int MAX_OF_BOUND = 40;
+    public static final int FIGURES_NUMBER = 5;
+    public static final String COLOR = "white";
+    private Random random = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        int randomNumber = random.nextInt(FIGURES_NUMBER);
+        switch (randomNumber) {
+            case 0:
+                return createCircle(colorSupplier.getRandomColor(), random.nextInt(MAX_OF_BOUND));
```

Comment:
```suggestion
                return createCircle();
```
randomize numbers inside those methods